SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -3492,6 +3488,18 @@ class CSolver {
   inline virtual void SetAitken_Relaxation(CGeometry *geometry,
                                            CConfig *config) { }
 
+  /*!
+   * \brief Loads the solution from the restart file.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] filename - Name of the restart file.
+   * \param[in] skipVars - Number of variables preceeding the solution.
+   */
+  void BasicLoadRestart(CGeometry *geometry,
+                        const CConfig *config,
+                        const string& filename,
+                        unsigned long skipVars);
```

Comment:
I created a method in CSolver to load very simple restarts, i.e. only the solution and no additional pre/post processing. This is now used by the disc adj solvers.